src/sql/workbench/services/profiler/browser/profilerService.ts

```diff
@@ -317,6 +318,18 @@ export class ProfilerService implements IProfilerService {
 		if (fileURIs?.length === 1) {
 			const fileURI = fileURIs[0];
 
+			try {
+				let fileSize = await (await fs.stat(fileURI.fsPath)).size;
+				console.log(fileSize);
```

Comment:
You can't use console in core code. If you want to log something use the logService (and if you do please add context so it's not just a random number showing up in the logs)

Reply:
Ah my bad, this is leftover from testing. Removing it.